### PR TITLE
feat(MobileSlider):added mobilr slider CSS

### PR DESCRIPTION
**Pull request by eyalkenig**

### CODE REVIEW

File: sass/core.scss [hunk 1]
```diff
@@ -3841,3 +3841,38 @@ div.yotpo-preloader-wrapper{
...
 
 /*End Animation */
+
+/* Mobile Slider */
+
+.yotpo-mobile-slider {
+  width:100%;
+  overflow-x: hidden;
+
+  .yotpo-mobile-holder {
```
Comment (livnekes) [inline sass/core.scss]: @eyalkenig didn't we rename the classes?

File: sass/shoppable_gallery.scss [hunk 2]
```diff
@@ -420,13 +416,13 @@
...
         color: #888888;
-        font-size: 18px;
+        font-size: 16px;
         font-weight: 400;
         letter-spacing: 0.13px;
         line-height: 44.619px;
         text-align: left;
         transform: scaleX(1.0083);
-        margin-left: 20px;
+        padding-left: 2.5%;
```
Comment (livnekes) [inline sass/shoppable_gallery.scss:440]: @eyalkenig why do we use %

File: sass/core.scss [hunk 3]
```diff
@@ -3841,3 +3841,38 @@ div.yotpo-preloader-wrapper{
...
+  -moz-transition: transform 0.3s ease-out;
+  -ms-transition: transform 0.3s ease-out;
+  -o-transition: transform 0.3s ease-out;
+  transition: transform 0.3s ease-out;
+}
+
+.yotpo-mobile-slider.ms-touch {
+  overflow-x: scroll;
+  overflow-y: hidden;
+  -ms-overflow-style: none;
```
Comment (livnekes) [inline sass/core.scss:3869]: @eyalkenig can you explain this rules

Comment (eyalkenig) [reply sass/core.scss:3869]: @j-bachman found this as a fix for IE problem

File: sass/shoppable_gallery.scss (hunk 2 above)
Comment (eyalkenig) [reply sass/shoppable_gallery.scss:440]: @livnekes up above there is .yotpo-shoppable-stats which has padding of 2.5%, and this element should be aligned to it.

File: sass/core.scss [hunk 4]
```diff
@@ -3841,3 +3841,35 @@ div.yotpo-preloader-wrapper{
...
+  overflow-x: hidden;
+
+  .yotpo-mobile-slide {
+    height: 100%;
+    float: left;
+    position: relative;
+    padding: 10px;
+  }
+}
+.yotpo-mobile-slide-animation {
```
Comment (livnekes) [inline sass/core.scss:3858]: @eyalkenig enter

**Pull request merged**